jiant/preprocess.py

```diff
@@ -190,15 +192,18 @@ def _build_embeddings(args, vocab, emb_file: str):
     log.info("\tBuilding embeddings from scratch.")
     word_v_size, unk_idx = vocab.get_vocab_size("tokens"), vocab.get_token_index(vocab._oov_token)
     embeddings = np.random.randn(word_v_size, args.d_word)
-    with io.open(
-        args.word_embs_file, "r", encoding="utf-8", newline="\n", errors="ignore"
-    ) as vec_fh:
-        for line in vec_fh:
-            word, vec = line.split(" ", 1)
-            idx = vocab.get_token_index(word)
-            if idx != unk_idx:
-                embeddings[idx] = np.array(list(map(float, vec.split())))
+
+    if "word_embs_file" in args and args.word_embs_file != "None":
```

Comment:
is it really a string "None" and not a `None` object ?

Reply:
Also, word_embs_file should always be in args

Reply:
This if statement should be unnecessary, since if input_module = glove or fasttext, then args.word_embs_file should always be a path. This should be checked somewhere in main, but if not, add an assert here.